.gitmodules

```diff
@@ -1,12 +1,3 @@
 [submodule "slam1/okvis"]
 	path = slam1/okvis
 	url = https://github.com/ILLIXR/okvis.git
```

Comment:
Why not remove okvis? That is, why not remove `.gitmodules` altogether?

Reply:
It's weird because the OKVIS code is not in the working tree, but the `.gitmodules` entry is still there. Normally, one should not manually edit the file, but I will try this time.